DigitalOffice/Services/FileService/Commands/Interfaces/IGetFileByIdCommand.cs

```diff
@@ -0,0 +1,10 @@
+﻿using LT.DigitalOffice.FileService.Models;
+using System;
+
+namespace LT.DigitalOffice.FileService.Commands.Interfaces
+{
+    public interface IGetFileByIdCommand
```

Comment:
XML documentation

Reply:
What do you mean?

Reply:
Most likely he offers you to add documentation to the interface.